2018-06-27 02:56:19.421 19605 DEBUG nova.osapi_compute.wsgi.server [req-d6944253-187f-42f1-ba47-278e134382d4 - - - - -] (19605) accepted ('10.0.20.21', 55504) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:56:19.444 19605 WARNING keystonemiddleware.auth_token [req-d6944253-187f-42f1-ba47-278e134382d4 - - - - -] Using the in-process token cache is deprecated as of the 4.2.0 release and may be removed in the 5.0.0 release or the 'O' development cycle. The in-process cache causes inconsistent results and high memory usage. When the feature is removed the auth_token middleware will not cache tokens by default which may result in performance issues. It is recommended to use  memcache for the auth_token token cache by setting the memcached_servers option.
2018-06-27 02:56:21.151 19605 DEBUG nova.api.openstack.wsgi [req-03741355-41c6-4f08-b19f-4dedafc85729 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Azione: 'create', metodo chiamata: <function version_select at 0x7efc4a014c80>, corpo: {"keypair": {"public_key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC3zBRwc99KrELjTPt/RD+bpFXaUE9UrW42L80skudWwp05Ma3UQkbxsTbFNZ6T2KJ9QNyVW6lwFC52Ks7MtDj4GiZxTmlUIvWxs0W5JqzHctkDz3nA+AFk2pGh+ZDDN8Eo+EBPQ1jI2e0pm/CV836jWZa2jqJQY03h3A0z4r/qxqMbWfCNlOdnUhLsEE+dbw2DurNqPGatWcjSYp5/qV5y9q1iPEaytF5wdjT2WlPpw4iNvmTAXoH3HWXnriMe2IFs2mEQjVOOBfZP6kCe5M4OSgfet88ZwxEI5VRE4XfUJRLL7g4rhXRkOcz6MTKeJx0jVix6AjcYdw7SrVYwWf2h root@localhost.localdomain\n", "name": "tempest-keypair-1530060918"}} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-27 02:56:21.242 19605 DEBUG oslo_db.sqlalchemy.engines [req-03741355-41c6-4f08-b19f-4dedafc85729 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:56:21.304 19605 DEBUG nova.quota [req-03741355-41c6-4f08-b19f-4dedafc85729 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Getting quotas for project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: set(['key_pairs']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-27 02:56:21.314 19605 DEBUG nova.quota [req-03741355-41c6-4f08-b19f-4dedafc85729 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Getting quotas for user f9ec6a476376492c8f3640e9c754f4e7 and project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: set(['key_pairs']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-27 02:56:21.441 19605 INFO nova.osapi_compute.wsgi.server [req-03741355-41c6-4f08-b19f-4dedafc85729 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "POST /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/os-keypairs HTTP/1.1" status: 200 len: 997 time: 2.0153821
2018-06-27 02:56:27.928 19601 DEBUG nova.osapi_compute.wsgi.server [req-d6944253-187f-42f1-ba47-278e134382d4 - - - - -] (19601) accepted ('10.0.20.21', 55518) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:56:27.936 19601 WARNING keystonemiddleware.auth_token [req-d6944253-187f-42f1-ba47-278e134382d4 - - - - -] Using the in-process token cache is deprecated as of the 4.2.0 release and may be removed in the 5.0.0 release or the 'O' development cycle. The in-process cache causes inconsistent results and high memory usage. When the feature is removed the auth_token middleware will not cache tokens by default which may result in performance issues. It is recommended to use  memcache for the auth_token token cache by setting the memcached_servers option.
2018-06-27 02:56:29.447 19601 DEBUG nova.api.openstack.wsgi [req-e383ee1b-e57b-44ed-83ea-88f162cdfec3 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<function version_select at 0x7efc4a014c80>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:56:29.475 19601 DEBUG oslo_db.sqlalchemy.engines [req-e383ee1b-e57b-44ed-83ea-88f162cdfec3 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:56:29.485 19601 WARNING oslo_db.sqlalchemy.utils [req-e383ee1b-e57b-44ed-83ea-88f162cdfec3 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Unique keys not in sort_keys. The sorting order may be unstable.
2018-06-27 02:56:29.500 19601 WARNING oslo_db.sqlalchemy.utils [req-e383ee1b-e57b-44ed-83ea-88f162cdfec3 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Unique keys not in sort_keys. The sorting order may be unstable.
2018-06-27 02:56:29.515 19601 INFO nova.osapi_compute.wsgi.server [req-e383ee1b-e57b-44ed-83ea-88f162cdfec3 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/os-keypairs HTTP/1.1" status: 200 len: 966 time: 1.5837491
2018-06-27 02:58:23.387 19599 DEBUG nova.osapi_compute.wsgi.server [req-d6944253-187f-42f1-ba47-278e134382d4 - - - - -] (19599) accepted ('10.0.20.21', 57936) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:58:23.401 19599 WARNING keystonemiddleware.auth_token [req-d6944253-187f-42f1-ba47-278e134382d4 - - - - -] Using the in-process token cache is deprecated as of the 4.2.0 release and may be removed in the 5.0.0 release or the 'O' development cycle. The in-process cache causes inconsistent results and high memory usage. When the feature is removed the auth_token middleware will not cache tokens by default which may result in performance issues. It is recommended to use  memcache for the auth_token token cache by setting the memcached_servers option.
2018-06-27 02:58:24.530 19603 DEBUG nova.api.openstack.wsgi [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a63fc90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:24.541 19603 DEBUG nova.compute.api [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Searching by: {u'changes-since': datetime.datetime(2018, 6, 27, 0, 53, 24, 340471, tzinfo=<FixedOffset u'+00:00' datetime.timedelta(0)>)} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 02:58:24.561 19603 DEBUG oslo_concurrency.lockutils [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:24.562 19603 DEBUG oslo_concurrency.lockutils [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:24.636 19603 DEBUG nova.compute.api [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 02:58:24.637 19603 DEBUG nova.compute.api [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 02:58:24.638 19603 DEBUG oslo_concurrency.lockutils [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:24.639 19603 DEBUG oslo_concurrency.lockutils [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:24.751 19603 DEBUG nova.policy [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'b5c7d57bbd1a48d5a86a8095ce57bfd4', 'roles': [u'ResellerAdmin', u'admin'], 'user_domain_id': u'default', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'6ce4dd91709c441d9f1b40034ea03af4', 'project_domain_id': u'default'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 02:58:24.763 19603 INFO nova.osapi_compute.wsgi.server [req-51ac7383-e89c-4249-b31e-e10f64b4bca7 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] 10.0.20.21 "GET /v2.1/6ce4dd91709c441d9f1b40034ea03af4/servers/detail?all_tenants=True&changes-since=2018-06-27T00%3A53%3A24.340471%2B00%3A00 HTTP/1.1" status: 200 len: 413 time: 0.6932120
2018-06-27 02:58:25.060 19599 DEBUG nova.api.openstack.wsgi [req-0e6f96f4-0541-422e-8aa2-46f62a12a3b6 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method FlavorsController.show of <nova.api.openstack.compute.flavors.FlavorsController object at 0x7efc4a978890>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:25.116 19599 DEBUG oslo_db.sqlalchemy.engines [req-0e6f96f4-0541-422e-8aa2-46f62a12a3b6 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:58:25.191 19599 INFO nova.api.openstack.wsgi [req-0e6f96f4-0541-422e-8aa2-46f62a12a3b6 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] HTTP exception thrown: Impossibile trovare la tipologia m1.tiny.
2018-06-27 02:58:25.192 19599 DEBUG nova.api.openstack.wsgi [req-0e6f96f4-0541-422e-8aa2-46f62a12a3b6 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Returning 404 to user: Impossibile trovare la tipologia m1.tiny. __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-27 02:58:25.195 19599 INFO nova.osapi_compute.wsgi.server [req-0e6f96f4-0541-422e-8aa2-46f62a12a3b6 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/flavors/m1.tiny HTTP/1.1" status: 404 len: 507 time: 1.8023930
2018-06-27 02:58:25.223 19599 DEBUG nova.api.openstack.wsgi [req-d8372485-6e88-4b96-9930-1d344ab922b4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method FlavorsController.index of <nova.api.openstack.compute.flavors.FlavorsController object at 0x7efc4a978090>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:25.324 19599 INFO nova.osapi_compute.wsgi.server [req-d8372485-6e88-4b96-9930-1d344ab922b4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/flavors HTTP/1.1" status: 200 len: 2106 time: 0.1129620
2018-06-27 02:58:25.365 19599 DEBUG nova.api.openstack.wsgi [req-49cc2875-01db-4c60-bf30-9bed6b0ca24a f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method FlavorsController.show of <nova.api.openstack.compute.flavors.FlavorsController object at 0x7efc4a978890>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:25.395 19599 INFO nova.osapi_compute.wsgi.server [req-49cc2875-01db-4c60-bf30-9bed6b0ca24a f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/flavors/1 HTTP/1.1" status: 200 len: 819 time: 0.0408578
2018-06-27 02:58:26.846 19599 DEBUG nova.api.openstack.wsgi [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Azione: 'create', metodo chiamata: <bound method ServersController.create of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a6d4a50>>, corpo: {"server": {"name": "tempest-INSTANCE_SAMPLE-1530060918", "imageRef": "57a73e2c-0852-4c8d-ac99-6c6af79162ca", "key_name": "tempest-keypair-1530060918", "flavorRef": "1", "max_count": 1, "min_count": 1, "networks": [{"uuid": "2a5f7645-767e-40d0-a92c-7516025abfa8"}], "security_groups": [{"name": "c629a6dd-9e53-41f3-bbd5-cfdae269ea90"}]}} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-27 02:58:26.945 19599 DEBUG oslo_concurrency.lockutils [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:26.947 19599 DEBUG oslo_concurrency.lockutils [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:26.975 19599 DEBUG oslo_db.sqlalchemy.engines [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:58:26.996 19599 DEBUG oslo_concurrency.lockutils [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:27.006 19599 DEBUG oslo_concurrency.lockutils [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.011s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:27.032 19599 DEBUG oslo_db.sqlalchemy.engines [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:58:27.095 19599 WARNING nova.image.glance [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] No protocol specified in for api_server 'http://10.0.20.21:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-06-27 02:58:29.312 19599 DEBUG nova.network.neutronv2.api [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] validate_networks() for [(u'2a5f7645-767e-40d0-a92c-7516025abfa8', None, None, None)] validate_networks /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1692
2018-06-27 02:58:30.173 19599 DEBUG nova.virt.hardware [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] emulator threads policy constraint: None get_emulator_threads_constraint /usr/lib/python2.7/site-packages/nova/virt/hardware.py:1289
2018-06-27 02:58:30.208 19599 DEBUG nova.quota [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Getting quotas for project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: ['metadata_items'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-27 02:58:30.220 19599 DEBUG nova.quota [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Getting quotas for user f9ec6a476376492c8f3640e9c754f4e7 and project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: ['metadata_items'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-27 02:58:30.274 19599 DEBUG nova.quota [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Getting quotas for project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: ['injected_files'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-27 02:58:30.283 19599 DEBUG nova.quota [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Getting quotas for user f9ec6a476376492c8f3640e9c754f4e7 and project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: ['injected_files'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-27 02:58:30.315 19599 DEBUG nova.quota [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Getting quotas for project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: ['injected_file_content_bytes', 'injected_file_path_bytes'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-27 02:58:30.324 19599 DEBUG nova.quota [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Getting quotas for user f9ec6a476376492c8f3640e9c754f4e7 and project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: ['injected_file_content_bytes', 'injected_file_path_bytes'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-27 02:58:30.653 19599 DEBUG nova.context [req-6cde665a-9ecd-4a25-beae-ac9714def93a - - - - -] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),10fe8ece-686d-4d33-8fcc-3d4b81191949(default) load_cells /usr/lib/python2.7/site-packages/nova/context.py:508
2018-06-27 02:58:30.655 19599 DEBUG oslo_concurrency.lockutils [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:30.655 19599 DEBUG oslo_concurrency.lockutils [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:30.656 19599 DEBUG oslo_concurrency.lockutils [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:30.657 19599 DEBUG oslo_concurrency.lockutils [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:30.690 19599 DEBUG nova.quota [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Getting quotas for project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-27 02:58:30.695 19599 DEBUG nova.quota [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Getting quotas for user f9ec6a476376492c8f3640e9c754f4e7 and project c6a8a1a4b15e43d18989d1fb9b1f8c2f. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-27 02:58:30.724 19599 DEBUG nova.compute.api [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Going to run 1 instances... _provision_instances /usr/lib/python2.7/site-packages/nova/compute/api.py:574
2018-06-27 02:58:30.787 19599 DEBUG nova.compute.api [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] block_device_mapping [BlockDeviceMapping(attachment_id=<?>,boot_index=0,connection_info=None,created_at=<?>,delete_on_termination=True,deleted=<?>,deleted_at=<?>,destination_type='local',device_name=None,device_type='disk',disk_bus=None,guest_format=None,id=<?>,image_id='57a73e2c-0852-4c8d-ac99-6c6af79162ca',instance=<?>,instance_uuid=<?>,no_device=False,snapshot_id=None,source_type='image',tag=None,updated_at=<?>,volume_id=None,volume_size=None)] _bdm_validate_set_size_and_instance /usr/lib/python2.7/site-packages/nova/compute/api.py:765
2018-06-27 02:58:31.092 19599 INFO nova.osapi_compute.wsgi.server [req-826ae4b7-e57a-4878-95af-db0fd6ef9685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "POST /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers HTTP/1.1" status: 202 len: 989 time: 4.2599211
2018-06-27 02:58:31.130 19599 DEBUG nova.api.openstack.wsgi [req-07a82fb7-bb79-48d4-9e71-d6510e8a2773 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:31.136 19599 DEBUG nova.compute.api [req-07a82fb7-bb79-48d4-9e71-d6510e8a2773 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 02:58:31.260 19599 DEBUG nova.policy [req-07a82fb7-bb79-48d4-9e71-d6510e8a2773 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 02:58:31.423 19599 INFO nova.osapi_compute.wsgi.server [req-07a82fb7-bb79-48d4-9e71-d6510e8a2773 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a HTTP/1.1" status: 200 len: 1857 time: 0.3129728
2018-06-27 02:58:31.539 19599 DEBUG nova.api.openstack.wsgi [req-73f6c7be-c342-4c89-bdbe-de1e4a43f1eb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method FlavorsController.show of <nova.api.openstack.compute.flavors.FlavorsController object at 0x7efc4a978890>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:31.589 19599 INFO nova.osapi_compute.wsgi.server [req-73f6c7be-c342-4c89-bdbe-de1e4a43f1eb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/flavors/1 HTTP/1.1" status: 200 len: 819 time: 0.0590439
2018-06-27 02:58:37.599 19598 DEBUG nova.osapi_compute.wsgi.server [req-2d02e56e-0107-4844-9f8e-fb896f1373c7 bd9839e84c0b4a919dc34283e9415cd3 f3aeffa936ea4fa6a6a3d11c34f61ddc - e1df930c942a4376b9d8615663bd31fe e1df930c942a4376b9d8615663bd31fe] (19598) accepted ('10.0.20.21', 58294) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:58:37.966 19598 INFO nova.osapi_compute.wsgi.server [req-115f6a98-bf6c-45f9-b381-5c367b75c8d1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f HTTP/1.1" status: 404 len: 382 time: 0.3635330
2018-06-27 02:58:37.992 19598 DEBUG nova.api.openstack.wsgi [req-055f53dc-2d77-46d4-8aa5-152fd07a6685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7efc4a9e7cd0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:37.995 19598 INFO nova.osapi_compute.wsgi.server [req-055f53dc-2d77-46d4-8aa5-152fd07a6685 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0187271
2018-06-27 02:58:38.391 19598 DEBUG nova.api.openstack.wsgi [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a689b90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:38.399 19598 DEBUG nova.compute.api [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', u'name': u'tempest-INSTANCE_SAMPLE-1530060918'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 02:58:38.414 19598 DEBUG oslo_concurrency.lockutils [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:38.415 19598 DEBUG oslo_concurrency.lockutils [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:38.514 19598 DEBUG nova.compute.api [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 02:58:38.515 19598 DEBUG nova.compute.api [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 02:58:38.516 19598 DEBUG oslo_concurrency.lockutils [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:38.519 19598 DEBUG oslo_concurrency.lockutils [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:38.611 19598 INFO nova.osapi_compute.wsgi.server [req-4a737f3d-5cc7-4fff-ba02-ea27df16d168 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers?name=tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 200 len: 786 time: 0.2309711
2018-06-27 02:58:38.654 19598 DEBUG nova.api.openstack.wsgi [req-c3519b59-79c9-447c-b426-47472b21685f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:38.659 19598 DEBUG nova.compute.api [req-c3519b59-79c9-447c-b426-47472b21685f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 02:58:38.701 19598 DEBUG oslo_concurrency.lockutils [req-c3519b59-79c9-447c-b426-47472b21685f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:38.702 19598 DEBUG oslo_concurrency.lockutils [req-c3519b59-79c9-447c-b426-47472b21685f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:38.874 19598 DEBUG nova.objects.instance [req-c3519b59-79c9-447c-b426-47472b21685f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lazy-loading 'services' on Instance uuid ab356d61-9cec-47af-965c-9d305e06796a obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-27 02:58:38.981 19598 DEBUG nova.policy [req-c3519b59-79c9-447c-b426-47472b21685f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 02:58:39.692 19598 INFO nova.osapi_compute.wsgi.server [req-c3519b59-79c9-447c-b426-47472b21685f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a HTTP/1.1" status: 200 len: 2359 time: 1.0540869
2018-06-27 02:58:48.062 19598 DEBUG nova.osapi_compute.wsgi.server [req-c3519b59-79c9-447c-b426-47472b21685f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] (19598) accepted ('10.0.20.21', 58430) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:58:48.878 19598 DEBUG nova.api.openstack.wsgi [req-edd1b18e-b5f3-462f-af4a-a09b6f62cab3 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Azione: 'create', metodo chiamata: <bound method ServerExternalEventsController.create of <nova.api.openstack.compute.server_external_events.ServerExternalEventsController object at 0x7efc4a845ed0>>, corpo: {"events": [{"tag": "347f2d0b-b1a2-4863-83a4-346c88dd9276", "name": "network-changed", "server_uuid": "ab356d61-9cec-47af-965c-9d305e06796a"}]} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-27 02:58:48.903 19598 DEBUG oslo_concurrency.lockutils [req-edd1b18e-b5f3-462f-af4a-a09b6f62cab3 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:48.905 19598 DEBUG oslo_concurrency.lockutils [req-edd1b18e-b5f3-462f-af4a-a09b6f62cab3 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:48.938 19598 INFO nova.api.openstack.compute.server_external_events [req-edd1b18e-b5f3-462f-af4a-a09b6f62cab3 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Creating event network-changed:347f2d0b-b1a2-4863-83a4-346c88dd9276 for instance ab356d61-9cec-47af-965c-9d305e06796a on localhost.localdomain
2018-06-27 02:58:49.217 19598 INFO nova.osapi_compute.wsgi.server [req-edd1b18e-b5f3-462f-af4a-a09b6f62cab3 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] 10.0.20.21 "POST /v2.1/6ce4dd91709c441d9f1b40034ea03af4/os-server-external-events HTTP/1.1" status: 200 len: 578 time: 1.1519859
2018-06-27 02:58:51.171 19604 DEBUG nova.osapi_compute.wsgi.server [req-fe92376a-ff92-4e44-b2a2-f4e278890f3c 5feafc9ff48747df82912c2055bf7429 f4a570c942f04ee3b556f3e6756b5a50 - default default] (19604) accepted ('10.0.20.21', 58460) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:58:51.776 19604 INFO nova.osapi_compute.wsgi.server [req-1682dfea-d2e8-4e47-a153-b6478ee93b0c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f HTTP/1.1" status: 404 len: 382 time: 0.5978899
2018-06-27 02:58:51.802 19604 DEBUG nova.api.openstack.wsgi [req-6fe8cee1-d0dc-4e2c-bb78-907aa91ac90d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7efc4a9e7cd0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:51.805 19604 INFO nova.osapi_compute.wsgi.server [req-6fe8cee1-d0dc-4e2c-bb78-907aa91ac90d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0149181
2018-06-27 02:58:52.328 19604 DEBUG nova.api.openstack.wsgi [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a689b90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:52.379 19604 DEBUG nova.compute.api [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', u'name': u'tempest-INSTANCE_SAMPLE-1530060918'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 02:58:52.553 19604 DEBUG oslo_db.sqlalchemy.engines [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:58:52.662 19604 DEBUG oslo_concurrency.lockutils [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:52.665 19604 DEBUG oslo_concurrency.lockutils [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:52.696 19604 DEBUG oslo_db.sqlalchemy.engines [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:58:52.832 19604 DEBUG nova.compute.api [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),10fe8ece-686d-4d33-8fcc-3d4b81191949(default) load_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:251
2018-06-27 02:58:52.834 19604 DEBUG nova.compute.api [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 02:58:52.835 19604 DEBUG nova.compute.api [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 02:58:52.837 19604 DEBUG oslo_concurrency.lockutils [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:52.843 19604 DEBUG oslo_concurrency.lockutils [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:52.869 19604 DEBUG oslo_db.sqlalchemy.engines [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:58:52.968 19604 INFO nova.osapi_compute.wsgi.server [req-6b70a4c5-7f53-4da7-b002-5712e25dba3f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers?name=tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 200 len: 786 time: 0.6554430
2018-06-27 02:58:53.014 19604 DEBUG nova.api.openstack.wsgi [req-f7651c82-3cd8-4d49-9a55-c17f55895b3e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:58:53.020 19604 DEBUG nova.compute.api [req-f7651c82-3cd8-4d49-9a55-c17f55895b3e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 02:58:53.054 19604 DEBUG oslo_concurrency.lockutils [req-f7651c82-3cd8-4d49-9a55-c17f55895b3e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:53.057 19604 DEBUG oslo_concurrency.lockutils [req-f7651c82-3cd8-4d49-9a55-c17f55895b3e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:53.066 19598 DEBUG nova.api.openstack.wsgi [req-1db0d8c2-2788-47e4-8a08-b199c3bc9687 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Azione: 'create', metodo chiamata: <bound method ServerExternalEventsController.create of <nova.api.openstack.compute.server_external_events.ServerExternalEventsController object at 0x7efc4a845ed0>>, corpo: {"events": [{"tag": "347f2d0b-b1a2-4863-83a4-346c88dd9276", "name": "network-vif-deleted", "server_uuid": "ab356d61-9cec-47af-965c-9d305e06796a"}]} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-27 02:58:53.090 19598 DEBUG oslo_concurrency.lockutils [req-1db0d8c2-2788-47e4-8a08-b199c3bc9687 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:58:53.092 19598 DEBUG oslo_concurrency.lockutils [req-1db0d8c2-2788-47e4-8a08-b199c3bc9687 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:58:53.151 19598 DEBUG nova.api.openstack.compute.server_external_events [req-1db0d8c2-2788-47e4-8a08-b199c3bc9687 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Unable to find a host for instance ab356d61-9cec-47af-965c-9d305e06796a. Dropping event network-vif-deleted create /usr/lib/python2.7/site-packages/nova/api/openstack/compute/server_external_events.py:138
2018-06-27 02:58:53.153 19598 INFO nova.api.openstack.wsgi [req-1db0d8c2-2788-47e4-8a08-b199c3bc9687 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] HTTP exception thrown: Non sono state rilevate istanze per alcun evento
2018-06-27 02:58:53.154 19598 DEBUG nova.api.openstack.wsgi [req-1db0d8c2-2788-47e4-8a08-b199c3bc9687 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] Returning 404 to user: Non sono state rilevate istanze per alcun evento __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-27 02:58:53.159 19598 INFO nova.osapi_compute.wsgi.server [req-1db0d8c2-2788-47e4-8a08-b199c3bc9687 4fbc52514c824825aeadd5a44fe1e4df 6ce4dd91709c441d9f1b40034ea03af4 - default default] 10.0.20.21 "POST /v2.1/6ce4dd91709c441d9f1b40034ea03af4/os-server-external-events HTTP/1.1" status: 404 len: 514 time: 0.1134782
2018-06-27 02:58:54.098 19604 DEBUG nova.policy [req-f7651c82-3cd8-4d49-9a55-c17f55895b3e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 02:58:56.178 19604 INFO nova.osapi_compute.wsgi.server [req-f7651c82-3cd8-4d49-9a55-c17f55895b3e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a HTTP/1.1" status: 200 len: 2267 time: 3.1827180
2018-06-27 02:59:07.049 19600 DEBUG nova.osapi_compute.wsgi.server [req-d6944253-187f-42f1-ba47-278e134382d4 - - - - -] (19600) accepted ('10.0.20.21', 59042) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:59:07.082 19600 WARNING keystonemiddleware.auth_token [req-d6944253-187f-42f1-ba47-278e134382d4 - - - - -] Using the in-process token cache is deprecated as of the 4.2.0 release and may be removed in the 5.0.0 release or the 'O' development cycle. The in-process cache causes inconsistent results and high memory usage. When the feature is removed the auth_token middleware will not cache tokens by default which may result in performance issues. It is recommended to use  memcache for the auth_token token cache by setting the memcached_servers option.
2018-06-27 02:59:09.112 19600 INFO nova.osapi_compute.wsgi.server [req-4ae06616-5cd5-47a4-8181-0283fef42477 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f HTTP/1.1" status: 404 len: 382 time: 2.0432670
2018-06-27 02:59:09.147 19600 DEBUG nova.api.openstack.wsgi [req-49ba8607-b3b8-4d8c-9e82-f7045f9ade49 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7efc4a9e7cd0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:09.154 19600 INFO nova.osapi_compute.wsgi.server [req-49ba8607-b3b8-4d8c-9e82-f7045f9ade49 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0205908
2018-06-27 02:59:09.584 19600 DEBUG nova.api.openstack.wsgi [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a689b90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:09.597 19600 DEBUG nova.compute.api [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', u'name': u'tempest-INSTANCE_SAMPLE-1530060918'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 02:59:09.687 19600 DEBUG oslo_db.sqlalchemy.engines [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:59:09.762 19600 DEBUG oslo_concurrency.lockutils [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:09.769 19600 DEBUG oslo_concurrency.lockutils [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.009s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:09.793 19600 DEBUG oslo_db.sqlalchemy.engines [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:59:09.921 19600 DEBUG nova.compute.api [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),10fe8ece-686d-4d33-8fcc-3d4b81191949(default) load_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:251
2018-06-27 02:59:09.922 19600 DEBUG nova.compute.api [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 02:59:09.923 19600 DEBUG nova.compute.api [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 02:59:09.926 19600 DEBUG oslo_concurrency.lockutils [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:09.933 19600 DEBUG oslo_concurrency.lockutils [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:09.954 19600 DEBUG oslo_db.sqlalchemy.engines [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:59:10.071 19600 INFO nova.osapi_compute.wsgi.server [req-ce8bb28f-0a14-4f8b-b255-d291446baa2f f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers?name=tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 200 len: 786 time: 0.4980919
2018-06-27 02:59:10.118 19600 DEBUG nova.api.openstack.wsgi [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:10.125 19600 DEBUG nova.compute.api [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 02:59:10.164 19600 DEBUG oslo_concurrency.lockutils [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:10.165 19600 DEBUG oslo_concurrency.lockutils [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:10.917 19600 DEBUG oslo_concurrency.lockutils [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:10.919 19600 DEBUG oslo_concurrency.lockutils [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:10.919 19600 DEBUG nova.objects.instance [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lazy-loading 'fault' on Instance uuid ab356d61-9cec-47af-965c-9d305e06796a obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-27 02:59:10.997 19600 DEBUG nova.policy [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 02:59:12.305 19600 INFO nova.osapi_compute.wsgi.server [req-86f8cac5-94a0-4f93-92de-d8932deff23e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a HTTP/1.1" status: 200 len: 3284 time: 2.2018440
2018-06-27 02:59:24.348 19601 DEBUG nova.osapi_compute.wsgi.server [req-e383ee1b-e57b-44ed-83ea-88f162cdfec3 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] (19601) accepted ('10.0.20.21', 59346) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:59:24.844 19601 INFO nova.osapi_compute.wsgi.server [req-f45b2ca5-f822-45cd-b4d7-0d9059d36c03 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f HTTP/1.1" status: 404 len: 382 time: 0.4935570
2018-06-27 02:59:24.870 19601 DEBUG nova.api.openstack.wsgi [req-5eef3b28-c9e3-4816-b386-c5b6e94705a8 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7efc4a9e7cd0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:24.874 19601 INFO nova.osapi_compute.wsgi.server [req-5eef3b28-c9e3-4816-b386-c5b6e94705a8 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0163529
2018-06-27 02:59:25.275 19601 DEBUG nova.api.openstack.wsgi [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a689b90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:25.280 19601 DEBUG nova.compute.api [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', u'name': u'tempest-INSTANCE_SAMPLE-1530060918'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 02:59:25.332 19601 DEBUG oslo_concurrency.lockutils [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:25.334 19601 DEBUG oslo_concurrency.lockutils [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:25.361 19601 DEBUG oslo_db.sqlalchemy.engines [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:59:25.478 19601 DEBUG nova.compute.api [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),10fe8ece-686d-4d33-8fcc-3d4b81191949(default) load_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:251
2018-06-27 02:59:25.479 19601 DEBUG nova.compute.api [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 02:59:25.480 19601 DEBUG nova.compute.api [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 02:59:25.481 19601 DEBUG oslo_concurrency.lockutils [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:25.484 19601 DEBUG oslo_concurrency.lockutils [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:25.504 19601 DEBUG oslo_db.sqlalchemy.engines [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 02:59:25.632 19601 INFO nova.osapi_compute.wsgi.server [req-1bf5718c-1b8d-42b4-97c6-d6eabe2980cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers?name=tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 200 len: 786 time: 0.3749070
2018-06-27 02:59:25.663 19601 DEBUG nova.api.openstack.wsgi [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:25.669 19601 DEBUG nova.compute.api [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 02:59:25.697 19601 DEBUG oslo_concurrency.lockutils [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:25.698 19601 DEBUG oslo_concurrency.lockutils [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:26.169 19601 DEBUG oslo_concurrency.lockutils [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:26.170 19601 DEBUG oslo_concurrency.lockutils [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:26.170 19601 DEBUG nova.objects.instance [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lazy-loading 'fault' on Instance uuid ab356d61-9cec-47af-965c-9d305e06796a obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-27 02:59:26.233 19601 DEBUG nova.policy [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 02:59:27.305 19601 INFO nova.osapi_compute.wsgi.server [req-29d2c631-b431-44cf-911d-329d083503b0 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a HTTP/1.1" status: 200 len: 3284 time: 1.6580369
2018-06-27 02:59:41.175 19604 DEBUG nova.osapi_compute.wsgi.server [req-f7651c82-3cd8-4d49-9a55-c17f55895b3e f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] (19604) accepted ('10.0.20.21', 59656) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 02:59:41.547 19604 INFO nova.osapi_compute.wsgi.server [req-6753faf4-6186-4d88-8cfe-659dc62d6283 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f HTTP/1.1" status: 404 len: 382 time: 0.3695631
2018-06-27 02:59:41.567 19604 DEBUG nova.api.openstack.wsgi [req-19775c28-fea7-4819-a990-1a3e21b7b1ab f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7efc4a9e7cd0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:41.569 19604 INFO nova.osapi_compute.wsgi.server [req-19775c28-fea7-4819-a990-1a3e21b7b1ab f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0110772
2018-06-27 02:59:41.792 19604 DEBUG nova.api.openstack.wsgi [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a689b90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:41.798 19604 DEBUG nova.compute.api [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', u'name': u'tempest-INSTANCE_SAMPLE-1530060918'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 02:59:41.813 19604 DEBUG oslo_concurrency.lockutils [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:41.814 19604 DEBUG oslo_concurrency.lockutils [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:41.860 19604 DEBUG nova.compute.api [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 02:59:41.861 19604 DEBUG nova.compute.api [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 02:59:41.862 19604 DEBUG oslo_concurrency.lockutils [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:41.862 19604 DEBUG oslo_concurrency.lockutils [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:41.924 19604 INFO nova.osapi_compute.wsgi.server [req-6b212934-8943-4ab3-aaf3-0cf6bd32940b f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers?name=tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 200 len: 786 time: 0.1412630
2018-06-27 02:59:41.953 19604 DEBUG nova.api.openstack.wsgi [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:41.957 19604 DEBUG nova.compute.api [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 02:59:41.970 19604 DEBUG oslo_concurrency.lockutils [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:41.971 19604 DEBUG oslo_concurrency.lockutils [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:42.064 19604 DEBUG oslo_concurrency.lockutils [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:42.065 19604 DEBUG oslo_concurrency.lockutils [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:42.065 19604 DEBUG nova.objects.instance [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lazy-loading 'fault' on Instance uuid ab356d61-9cec-47af-965c-9d305e06796a obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-27 02:59:42.100 19604 DEBUG nova.policy [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 02:59:42.519 19604 INFO nova.osapi_compute.wsgi.server [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a HTTP/1.1" status: 200 len: 3284 time: 0.5781639
2018-06-27 02:59:46.485 19598 DEBUG nova.api.openstack.wsgi [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a63fc90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 02:59:46.498 19598 DEBUG nova.compute.api [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Searching by: {u'changes-since': datetime.datetime(2018, 6, 27, 0, 54, 47, 22567, tzinfo=<FixedOffset u'+00:00' datetime.timedelta(0)>)} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 02:59:46.512 19598 DEBUG oslo_concurrency.lockutils [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:46.512 19598 DEBUG oslo_concurrency.lockutils [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:46.576 19598 DEBUG nova.compute.api [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 02:59:46.577 19598 DEBUG nova.compute.api [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 02:59:46.578 19598 DEBUG oslo_concurrency.lockutils [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:46.578 19598 DEBUG oslo_concurrency.lockutils [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:46.716 19598 DEBUG oslo_concurrency.lockutils [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 02:59:46.717 19598 DEBUG oslo_concurrency.lockutils [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 02:59:46.735 19598 DEBUG nova.policy [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'b5c7d57bbd1a48d5a86a8095ce57bfd4', 'roles': [u'ResellerAdmin', u'admin'], 'user_domain_id': u'default', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'6ce4dd91709c441d9f1b40034ea03af4', 'project_domain_id': u'default'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 02:59:47.323 19598 INFO nova.osapi_compute.wsgi.server [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] 10.0.20.21 "GET /v2.1/6ce4dd91709c441d9f1b40034ea03af4/servers/detail?all_tenants=True&changes-since=2018-06-27T00%3A54%3A47.022567%2B00%3A00 HTTP/1.1" status: 200 len: 2896 time: 1.1214879
2018-06-27 03:00:11.930 19598 DEBUG nova.osapi_compute.wsgi.server [req-a55b8b23-d839-4930-95c8-af09167eddd9 b5c7d57bbd1a48d5a86a8095ce57bfd4 6ce4dd91709c441d9f1b40034ea03af4 - default default] (19598) accepted ('10.0.20.21', 60508) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 03:00:12.280 19598 DEBUG nova.api.openstack.wsgi [req-a1a18178-8450-497e-901a-dcf15e2adb87 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:00:12.284 19598 DEBUG nova.compute.api [req-a1a18178-8450-497e-901a-dcf15e2adb87 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Failed to fetch instance by id tempest-INSTANCE_SAMPLE-1530060918 get /usr/lib/python2.7/site-packages/nova/compute/api.py:1341
2018-06-27 03:00:12.284 19598 INFO nova.api.openstack.wsgi [req-a1a18178-8450-497e-901a-dcf15e2adb87 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] HTTP exception thrown: Impossibile trovare l'istanza tempest-INSTANCE_SAMPLE-1530060918.
2018-06-27 03:00:12.285 19598 DEBUG nova.api.openstack.wsgi [req-a1a18178-8450-497e-901a-dcf15e2adb87 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Returning 404 to user: Impossibile trovare l'istanza tempest-INSTANCE_SAMPLE-1530060918. __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-27 03:00:12.286 19598 INFO nova.osapi_compute.wsgi.server [req-a1a18178-8450-497e-901a-dcf15e2adb87 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 404 len: 532 time: 0.3542840
2018-06-27 03:00:12.307 19598 DEBUG nova.api.openstack.wsgi [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a689b90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:00:12.311 19598 DEBUG nova.compute.api [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', u'name': u'tempest-INSTANCE_SAMPLE-1530060918'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 03:00:12.328 19598 DEBUG oslo_concurrency.lockutils [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:12.329 19598 DEBUG oslo_concurrency.lockutils [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:12.374 19598 DEBUG nova.compute.api [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 03:00:12.375 19598 DEBUG nova.compute.api [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 03:00:12.376 19598 DEBUG oslo_concurrency.lockutils [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:12.377 19598 DEBUG oslo_concurrency.lockutils [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:12.427 19598 INFO nova.osapi_compute.wsgi.server [req-e9c94b6f-e1ee-4f61-b050-2f04a87adf34 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers?name=tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 200 len: 784 time: 0.1281829
2018-06-27 03:00:12.446 19598 DEBUG nova.api.openstack.wsgi [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:00:12.449 19598 DEBUG nova.compute.api [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 03:00:12.457 19598 DEBUG oslo_concurrency.lockutils [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:12.458 19598 DEBUG oslo_concurrency.lockutils [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:12.528 19598 DEBUG oslo_concurrency.lockutils [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:12.529 19598 DEBUG oslo_concurrency.lockutils [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:12.530 19598 DEBUG nova.objects.instance [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lazy-loading 'fault' on Instance uuid ab356d61-9cec-47af-965c-9d305e06796a obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-27 03:00:12.570 19598 DEBUG nova.policy [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 03:00:13.263 19598 INFO nova.osapi_compute.wsgi.server [req-e32aa4e6-eb54-4a27-abd2-b4b6a315b536 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a HTTP/1.1" status: 200 len: 2893 time: 0.8258390
2018-06-27 03:00:13.283 19598 DEBUG nova.api.openstack.wsgi [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Azione: 'action', metodo chiamata: <function version_select at 0x7efc4a5b7140>, corpo: {"addFloatingIp": {"address": "172.24.4.6"}} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-27 03:00:13.288 19598 DEBUG nova.compute.api [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 03:00:13.295 19598 DEBUG oslo_concurrency.lockutils [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:13.296 19598 DEBUG oslo_concurrency.lockutils [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:13.355 19598 WARNING nova.api.openstack.compute.floating_ips [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Info cache is InstanceInfoCache(created_at=2018-06-27T00:58:35Z,deleted=False,deleted_at=None,instance_uuid=ab356d61-9cec-47af-965c-9d305e06796a,network_info=NetworkModel(),updated_at=2018-06-27T00:58:53Z) during associate with no nw_info cache
2018-06-27 03:00:13.356 19598 INFO nova.api.openstack.wsgi [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] HTTP exception thrown: Instance network is not ready yet
2018-06-27 03:00:13.357 19598 DEBUG nova.api.openstack.wsgi [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Returning 400 to user: Instance network is not ready yet __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-27 03:00:13.358 19598 INFO nova.osapi_compute.wsgi.server [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "POST /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a/action HTTP/1.1" status: 400 len: 499 time: 0.0835528
2018-06-27 03:00:22.635 19604 DEBUG nova.osapi_compute.wsgi.server [req-dad6381c-1c45-4afe-bee0-08f956cb7ef2 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] (19604) accepted ('10.0.20.21', 60528) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 03:00:23.051 19604 INFO nova.osapi_compute.wsgi.server [req-2c4c9d00-9da6-4493-889b-92b049742ee5 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f HTTP/1.1" status: 404 len: 382 time: 0.4143951
2018-06-27 03:00:23.070 19604 DEBUG nova.api.openstack.wsgi [req-cf86b610-83b4-4390-ab83-c71851c0c0dc f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7efc4a9e7cd0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:00:23.072 19604 INFO nova.osapi_compute.wsgi.server [req-cf86b610-83b4-4390-ab83-c71851c0c0dc f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0114789
2018-06-27 03:00:23.352 19604 DEBUG nova.api.openstack.wsgi [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a63fc90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:00:23.357 19604 DEBUG nova.compute.api [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 03:00:23.372 19604 DEBUG oslo_concurrency.lockutils [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:23.373 19604 DEBUG oslo_concurrency.lockutils [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:23.456 19604 DEBUG nova.compute.api [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 03:00:23.457 19604 DEBUG nova.compute.api [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 03:00:23.458 19604 DEBUG oslo_concurrency.lockutils [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:23.459 19604 DEBUG oslo_concurrency.lockutils [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:23.610 19604 DEBUG oslo_concurrency.lockutils [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:23.611 19604 DEBUG oslo_concurrency.lockutils [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:23.625 19604 DEBUG nova.policy [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 03:00:24.116 19604 INFO nova.osapi_compute.wsgi.server [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/detail HTTP/1.1" status: 200 len: 3287 time: 0.7734308
2018-06-27 03:00:41.657 19604 DEBUG nova.osapi_compute.wsgi.server [req-fface3b6-40fd-4123-8299-ba460e644954 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] (19604) accepted ('10.0.20.21', 60900) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 03:00:42.113 19604 DEBUG nova.api.openstack.wsgi [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a63fc90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:00:42.120 19604 DEBUG nova.compute.api [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 03:00:42.133 19604 DEBUG oslo_concurrency.lockutils [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:42.134 19604 DEBUG oslo_concurrency.lockutils [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:42.190 19604 DEBUG nova.compute.api [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 03:00:42.191 19604 DEBUG nova.compute.api [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 03:00:42.192 19604 DEBUG oslo_concurrency.lockutils [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:42.192 19604 DEBUG oslo_concurrency.lockutils [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:42.333 19604 DEBUG oslo_concurrency.lockutils [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:42.335 19604 DEBUG oslo_concurrency.lockutils [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:42.351 19604 DEBUG nova.policy [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 03:00:42.989 19604 INFO nova.osapi_compute.wsgi.server [req-c14175ee-9c12-4b27-8ce8-88006725ce7c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/detail HTTP/1.1" status: 200 len: 2896 time: 1.3278370
2018-06-27 03:00:53.779 19605 DEBUG nova.osapi_compute.wsgi.server [req-03741355-41c6-4f08-b19f-4dedafc85729 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] (19605) accepted ('10.0.20.21', 60940) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 03:00:54.237 19605 DEBUG nova.api.openstack.wsgi [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a63fc90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:00:54.243 19605 DEBUG nova.compute.api [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 03:00:54.273 19605 DEBUG oslo_concurrency.lockutils [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:54.275 19605 DEBUG oslo_concurrency.lockutils [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:54.293 19605 DEBUG oslo_db.sqlalchemy.engines [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 03:00:54.429 19605 DEBUG nova.compute.api [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),10fe8ece-686d-4d33-8fcc-3d4b81191949(default) load_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:251
2018-06-27 03:00:54.431 19605 DEBUG nova.compute.api [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 03:00:54.432 19605 DEBUG nova.compute.api [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 03:00:54.433 19605 DEBUG oslo_concurrency.lockutils [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:54.438 19605 DEBUG oslo_concurrency.lockutils [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:54.458 19605 DEBUG oslo_db.sqlalchemy.engines [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-27 03:00:54.948 19605 DEBUG oslo_concurrency.lockutils [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:00:54.949 19605 DEBUG oslo_concurrency.lockutils [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:00:54.972 19605 DEBUG nova.policy [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 03:00:55.879 19605 INFO nova.osapi_compute.wsgi.server [req-cdbfefd5-d54b-4520-9614-73947caf0cf1 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/detail HTTP/1.1" status: 200 len: 2896 time: 2.0971808
2018-06-27 03:01:09.370 19599 DEBUG nova.osapi_compute.wsgi.server [req-73f6c7be-c342-4c89-bdbe-de1e4a43f1eb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] (19599) accepted ('10.0.20.21', 33512) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 03:01:09.835 19599 DEBUG nova.api.openstack.wsgi [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a63fc90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:01:09.840 19599 DEBUG nova.compute.api [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 03:01:09.856 19599 DEBUG oslo_concurrency.lockutils [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:01:09.857 19599 DEBUG oslo_concurrency.lockutils [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:01:09.972 19599 DEBUG nova.compute.api [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),10fe8ece-686d-4d33-8fcc-3d4b81191949(default) load_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:251
2018-06-27 03:01:09.972 19599 DEBUG nova.compute.api [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 03:01:09.973 19599 DEBUG nova.compute.api [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 03:01:09.974 19599 DEBUG oslo_concurrency.lockutils [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:01:09.975 19599 DEBUG oslo_concurrency.lockutils [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:01:10.106 19599 DEBUG oslo_concurrency.lockutils [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:01:10.107 19599 DEBUG oslo_concurrency.lockutils [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:01:10.121 19599 DEBUG nova.policy [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 03:01:10.563 19599 INFO nova.osapi_compute.wsgi.server [req-6d8881c9-dcf6-4d5c-a4aa-b233fc7428e4 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/detail HTTP/1.1" status: 200 len: 2896 time: 1.1891861
2018-06-27 03:01:16.784 19598 DEBUG nova.osapi_compute.wsgi.server [req-37352d12-8307-489f-9c95-af1f1a17b192 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] (19598) accepted ('10.0.20.21', 33528) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-27 03:01:17.263 19598 DEBUG nova.api.openstack.wsgi [req-37f3bc52-02a7-48f7-8b60-b3dadf3cc9cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:01:17.268 19598 DEBUG nova.compute.api [req-37f3bc52-02a7-48f7-8b60-b3dadf3cc9cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Failed to fetch instance by id tempest-INSTANCE_SAMPLE-1530060918 get /usr/lib/python2.7/site-packages/nova/compute/api.py:1341
2018-06-27 03:01:17.270 19598 INFO nova.api.openstack.wsgi [req-37f3bc52-02a7-48f7-8b60-b3dadf3cc9cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] HTTP exception thrown: Impossibile trovare l'istanza tempest-INSTANCE_SAMPLE-1530060918.
2018-06-27 03:01:17.271 19598 DEBUG nova.api.openstack.wsgi [req-37f3bc52-02a7-48f7-8b60-b3dadf3cc9cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Returning 404 to user: Impossibile trovare l'istanza tempest-INSTANCE_SAMPLE-1530060918. __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-27 03:01:17.278 19598 INFO nova.osapi_compute.wsgi.server [req-37f3bc52-02a7-48f7-8b60-b3dadf3cc9cb f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 404 len: 532 time: 0.4920268
2018-06-27 03:01:17.323 19598 DEBUG nova.api.openstack.wsgi [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a689b90>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:01:17.334 19598 DEBUG nova.compute.api [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Searching by: {'deleted': False, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', u'name': u'tempest-INSTANCE_SAMPLE-1530060918'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:1352
2018-06-27 03:01:17.365 19598 DEBUG oslo_concurrency.lockutils [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:01:17.366 19598 DEBUG oslo_concurrency.lockutils [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:01:17.486 19598 DEBUG nova.compute.api [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1463
2018-06-27 03:01:17.486 19598 DEBUG nova.compute.api [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Listing 1000 instances in cell 10fe8ece-686d-4d33-8fcc-3d4b81191949(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:1465
2018-06-27 03:01:17.488 19598 DEBUG oslo_concurrency.lockutils [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:01:17.488 19598 DEBUG oslo_concurrency.lockutils [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:01:17.597 19598 INFO nova.osapi_compute.wsgi.server [req-cd2f1b60-eb63-4a82-a3b5-0ed09ea7f16c f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers?name=tempest-INSTANCE_SAMPLE-1530060918 HTTP/1.1" status: 200 len: 784 time: 0.2910440
2018-06-27 03:01:17.627 19598 DEBUG nova.api.openstack.wsgi [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7efc4a566050>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-27 03:01:17.633 19598 DEBUG nova.compute.api [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 03:01:17.658 19598 DEBUG oslo_concurrency.lockutils [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:01:17.661 19598 DEBUG oslo_concurrency.lockutils [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:01:17.906 19598 DEBUG oslo_concurrency.lockutils [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:01:17.907 19598 DEBUG oslo_concurrency.lockutils [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:01:17.908 19598 DEBUG nova.objects.instance [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lazy-loading 'fault' on Instance uuid ab356d61-9cec-47af-965c-9d305e06796a obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-27 03:01:17.993 19598 DEBUG nova.policy [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'f9ec6a476376492c8f3640e9c754f4e7', 'roles': [u'admin'], 'user_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'c6a8a1a4b15e43d18989d1fb9b1f8c2f', 'project_domain_id': u'3bf7ccf4010644abbf0558a90e6c574f'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-27 03:01:18.793 19598 INFO nova.osapi_compute.wsgi.server [req-f0aba7d5-8244-46ec-aff9-1a5a5d54720d f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "GET /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a HTTP/1.1" status: 200 len: 2893 time: 1.1807809
2018-06-27 03:01:19.527 19598 DEBUG nova.api.openstack.wsgi [req-90c96bb4-7659-4d0d-9dfc-849767115b29 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Azione: 'create', metodo chiamata: <bound method VolumeAttachmentController.create of <nova.api.openstack.compute.volumes.VolumeAttachmentController object at 0x7efc4a475810>>, corpo: {"volumeAttachment": {"device": "/dev/vdb", "volumeId": "1ea47ff3-7a81-4294-b579-38bcd8c8ee82"}} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-27 03:01:19.537 19598 DEBUG nova.compute.api [req-90c96bb4-7659-4d0d-9dfc-849767115b29 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] [instance: ab356d61-9cec-47af-965c-9d305e06796a] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:1338
2018-06-27 03:01:19.551 19598 DEBUG oslo_concurrency.lockutils [req-90c96bb4-7659-4d0d-9dfc-849767115b29 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 03:01:19.552 19598 DEBUG oslo_concurrency.lockutils [req-90c96bb4-7659-4d0d-9dfc-849767115b29 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Lock "10fe8ece-686d-4d33-8fcc-3d4b81191949" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 03:01:19.648 19598 INFO nova.api.openstack.wsgi [req-90c96bb4-7659-4d0d-9dfc-849767115b29 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] HTTP exception thrown: Impossibile 'attach_volume' l'istanza ab356d61-9cec-47af-965c-9d305e06796a mentre si trova in vm_state error
2018-06-27 03:01:19.650 19598 DEBUG nova.api.openstack.wsgi [req-90c96bb4-7659-4d0d-9dfc-849767115b29 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] Returning 409 to user: Impossibile 'attach_volume' l'istanza ab356d61-9cec-47af-965c-9d305e06796a mentre si trova in vm_state error __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-27 03:01:19.654 19598 INFO nova.osapi_compute.wsgi.server [req-90c96bb4-7659-4d0d-9dfc-849767115b29 f9ec6a476376492c8f3640e9c754f4e7 c6a8a1a4b15e43d18989d1fb9b1f8c2f - 3bf7ccf4010644abbf0558a90e6c574f 3bf7ccf4010644abbf0558a90e6c574f] 10.0.20.21 "POST /v2.1/c6a8a1a4b15e43d18989d1fb9b1f8c2f/servers/ab356d61-9cec-47af-965c-9d305e06796a/os-volume_attachments HTTP/1.1" status: 409 len: 580 time: 0.1384380
